lib/services/user_test.go

```diff
@@ -287,6 +289,54 @@ func TestOIDCMapping(t *testing.T) {
 	}
 }
 
+
+/*
```

Comment:
nit: Remove the comment with benchmark results - it's better to run it again if necessary.

Reply:
Done in 7eaa6ea9df5f4a18b2c55f85c85e5e19d9628d99.